src/api/index.js

```diff
@@ -316,3 +316,20 @@ export const fetchMergePullRequest = (
     })
   );
 };
+
+export const fetchSubmitNewIssue = (
```

Comment:
I agree with you but I was inspired by the `fetchPostIssueComment` ([L182](https://github.com/gitpoint/git-point/blob/master/src/api/index.js#L182)) function name. For consistency questions I chose to keep the `fetch` keyword :)